SawtoothSigning/Context.swift

```diff
@@ -24,7 +24,7 @@ public protocol Context {
 
     /**
         Create a signature by signing the bytes.
-     
+
```

Comment:
Should the methods in Context also have information on the errors they throw?

Reply:
I didn't document the errors in the protocols because the protocol definition doesn't provide any information about the type of errors that can be thrown. Different implementations of the protocols could throw differing errors.